src/qmake/debug.pri

```diff
@@ -2,7 +2,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-debug {
+CONFIG(debug, debug|release) {
```

Comment:
We are switching to cmake. Can you please use cmake instead? In this way you probably do not have this issue.

Reply:
I can’t yet, because the current release v2.8.0 doesn’t have it, and Qt6 is not yet in nixpkgs (NixOS/nixpkgs#169296). When you cut a new release and Qt6 enters nixpkgs I will switch.

But meanwhile, this code is still present and should be fixed, because unexpectedly exposing QML debugging on [port 1234 of all interfaces](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/src/commands/commandui.cpp#L174-L175) is [insecure](https://doc.qt.io/qt-6/qtquick-debugging.html#qml-debugging-infrastructure).